src/App.tsx

```diff
@@ -1,10 +1,20 @@
 import "./App.css";
+import Practice1 from "./components/practice1";
 
 function App() {
 	return (
-		<header className="bg-[#94A3B8] text-center p-[20px] text-4xl text-[#F9FAFB]">
+		<div>
+		<div className="box-border pb-80 ml-60 mr-60 mt-20 h-200 w-104 border-4 ..."> 
```

Comment:
`ml-60 mr-60` is the same as `mx-60` I think the latter one is better in terms of readability.

Reply:
Understood